analyzer/codechecker_analyzer/analyzer_env.py

```diff
@@ -77,3 +77,15 @@ def extend_analyzer_cmd_with_resource_dir(analyzer_cmd,
 
     analyzer_cmd.extend(['-nobuiltininc',
                          '-isystem', resource_inc])
+
```

Comment:
maybe we could rename this file `analyzer/codechecker_analyzer/analyzer_env.py` to `analyzer/codechecker_analyzer/env.py`, what do you think?